vlf.el

```diff
@@ -173,7 +173,7 @@ Return newly created buffer."
     (set-visited-file-name file)
     (set-buffer-modified-p nil)
     (if (or minimal (file-remote-p file))
-        (set (make-local-variable 'vlf-batch-size) 1024))
+        (set (make-local-variable 'vlf-batch-size) vlf-batch-size-remote))
```

Comment:
Split the minimal and remote-file-p cases.  Minimal should stay 1024.
It's meant for internal purposes (like batch operations).